cluster/ci/config/prow/config.yaml

```diff
@@ -343,9 +344,6 @@ presubmits:
     labels:
       master: ci.openshift.redhat.com
     always_run: false
-    # TEMPORARY until this job is fixed to work
-    skip_report: true
-    run_if_changed: '^(pkg/cmd/server/bootstrappolicy)|^(pkg/oc/bootstrap/docker/openshift)|^(cmd/service-catalog)'
```

Comment:
Why is this not a run_if_changed job anymore?

Reply:
This show my lack of understanding here, but does prow only initiate jobs based on origin commits? The run_if_changed is removed because those directories will no longer exist once the catalog code is switched to use an outside repo. This PR may provide clarification: https://github.com/openshift/aos-cd-jobs/pull/1278